index.html

```diff
@@ -0,0 +1,75 @@
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
+            <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+        </section>
+        <div class="contain">
+            <div class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
+                <h2 class="item2">Software Developer</h2>
+                <div class="literature">
+                    <p class="item3">I can help you build a product , feature or website Look through some of my work and experience!<br> If you like what you see and have a project you need coded,<br> don’t hestiate to contact me.</p>
+                </div>
+            </div>
+        </div>
+        <header class="border-bottom">
+            <ul class="skills-list">
+                <li class="language">
+                    <h3 class="skill-title">Language</h3>
+                    <ul class="sublist">
+                        <li class="skill">JavaScript</li>
+                        <li class="skill">Ruby</li>
+                        <li class="skill">Html</li>
+                        <li class="skill">Css</li>
+                    </ul>
+                </li>
+                <li class="frameworks">
+                    <h3 class="skill-title">Frameworks</h3>
+                    <ul class="sublist">
+                        <li class="skill">Bootstrap</li>
+                        <li class="skill">Ruby on Rails</li>
+                        <li class="skill">RSPec</li>
+                        <li class="skill">Capy Bary</li>
+                        <li class="skill">Selenium</li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <h3 class="skill-title">Skills</h3>
+                    <ul class="sublist">
+                        <li class="skill">Codekit</li>
+                        <li class="skill">Github</li>
+                        <li class="skill">Codpen</li>
+                        <li class="skill">Gitlab</li>
+                        <li class="skill">Terminal</li>
+                    </ul>
+                </li>
+            </ul>
+        </header>
+        <hr/>
+        <ul class="icones">
+            <li><img src="images/icones/sector.png" alt="icon1"></li>
+            <li><img src="images/icones/Vecto.png" alt=" icon2"></li>
+            <li><img src="images/icones/Vector.png" alt="icon3"></li>
+            <li><img src="images/icones/Vector.svg" alt="icon4"></li>
+            <li><img src="images/icones/Linkedin.png" alt="linkedinicon"></li>
+        </ul>
+        <div>
+            <div class="eli18"><img src="images/icones/Ellipse18.svg" alt=" img ackround "></div>
+            <div class="eli19"><img src="images/icones/Ellipse19.svg" alt="img backgound "></div>
+        </div>
```

Comment:
- [ ] Since these are decorator images and they don't have significance in relation to the Portfolio content, they should be added as CSS background images instead. That way, screen readers won't bother reading them and make your website more accessible as a result. Kindly refer [this](https://www.w3schools.com/cssref/pr_background-image.asp) on how to add css background images.